usecases/cluster/transactions_broadcast.go

```diff
@@ -91,3 +128,13 @@ func (t *TxBroadcaster) BroadcastCommitTransaction(ctx context.Context, tx *Tran
 
 	return eg.Wait()
 }
+
+func copyTx(in *Transaction) *Transaction {
```

Comment:
`copyTx()` might not return a deep copy of the Transaction if payload has a value of pointer type.
In case a deep copy is not required this function would be equivalent to:
```
func copyTx(in *Transaction) *Transaction {
 copy := *in
 return &copy
}
```

Reply:
Great point, I don't think a deep copy is required, so your code is much more concise. I'll update it.